### ISSUE TITLE
cast error on new EVM deployment

### ISSUE THREAD

**Issue by jran3228** [ProjectOpenSea/seaport#1021]
`cast publish --rpc-url ${RPC_URL} 0xf87e8085174876e800830186a08080ad601f80600e600039806000f350fe60003681823780368234f58015156014578182fd5b80825250506014600cf31ba02222222222222222222222222222222222222222222222222222222222222222a02222222222222222222222222222222222222222222222222222222222222222`

Using this command, with RPC_URL = "new evm non deployed chain RPC url" brings up this error.  This chain has no seaport deployments.  

`(code: -32603, message: invalid key format, data: None) `

Since this is an RPC with no seaport deployments(including KEYLESS_CREATE2_DEPLOYER_ADDRESS, ETC),  would those need to be deployed(Are those deployment CREATE2 factory contracts available?).  

Assuming otherwise, if just using the CREATE strategy the deployment order should be(without CREATE2 factories):

- Conduit Controler Contract (STEP 1)
- Seaport Contract(STEP 2, which has input for Conduit Controller address). 

Thank you

**Comment by vishal590** [comment]
opensea has discord server ?

**Comment by ryanio** [comment]
Can you try the latest instructions in https://github.com/ProjectOpenSea/seaport/blob/main/docs/Deployment.md and let us know if you're still having problems?

**Comment by restlessronin** [comment]
@ryanio I'm trying to deploy seaport on the polygon zkevm testnet. The deployment instructions you provided seem to work. I say seem to, because it appears someone else had already run them. I managed to confirm (step 3 of deployment) that the seaport name was returned.

I'm running into problems in the verification. When I try to navigate to the 1.1 version tag and verify 'ConduitController', using
```
npx hardhat verify --network verificationNetwork "0x00000000F9490004C11Cef243f5400493c00Ad63"
```
I get the error
```
Error HH303: Unrecognized task verify
```
This seems related to an earlier version of hardhat being used in that release.

I get a different kind of error when I navigate to the 1.5 version. The build does not complete with 'warm' errors.

 Bottom line, I am unable to complete the verification of either ConduitController or Seaport.

The latest version on the other hand builds fine.

**Comment by restlessronin** [comment]
@ryanio I have a somewhat related question. Assuming that these contracts are "correct", I would still need the canonical conduit (https://github.com/ProjectOpenSea/seaport/discussions/580#discussioncomment-3172146) in order to run the protocol.

I can find no instructions on how to deploy the canonical conduit to a new chain.
